README.md

```diff
@@ -115,6 +115,95 @@ grunt.registerTask('standards', [ 'markdownlint' ]);
 
 TODO: fill in details and examples here.
 
+
+### Release Process
+
+#### Configuration
+
+   1. Ensure that the project's `package.json` file has a
+      [`repository.url`][package-json-repo-url] field with the URL to the canonical repo
+      for the project in its git hosting solution, e.g.
+      <https://github.com/silvermine/event-emitter.git> for the
+      [@silvermine/event-emitter](https://github.com/silvermine/event-emitter) project.
+      * This is necessary because conventional-changelog needs to know the URL to the git
+         hosting solution so that it can make links to "compare URLs" in the CHANGELOG
+   1. Add the following NPM scripts to the project's `package.json` file:
+
+      ```json
+      "release:prep-changelog": "./node_modules/@silvermine/standardization/scripts/release.sh prep-changelog",
+      "release:finalize": "./node_modules/@silvermine/standardization/scripts/release.sh finalize"
```

Comment:
Looks like these would need to be updated to point to the Node versions of these scripts, if this implementation were to be accepted.

Reply:
🤦‍♂️  Good catch